examples/terra-contract/src/msg.rs

```diff
@@ -25,7 +25,7 @@ pub enum ExecuteMsg {}
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum QueryMsg {
-    FetchPrice,
+    FetchPrice {},
```

Comment:
Is the empty `{}` form better than the previous style?

Reply:
No not at all. But the reason is that json message for the previous one becomes `"fetch_price"` and terra finder UI is stupid and doesn't work with this, 

It basically removes the double quotes :( 

```https://bombay-fcd.terra.dev/wasm/contracts/terra16h868tx50d3w37ry7c5lzzg648f7yetu39p5pd/store?query_msg=fetch_price```

And there is no way around it to fix it.